docs/guide/_6_buffers.md

```diff
@@ -0,0 +1,27 @@
+# Python Buffer Protocol
+
+Python objects implementing the [Buffer Protocol](https://docs.python.org/3/c-api/buffer.html#) can be used with zero copy.
+
+```zig
+--8<-- "example/buffers.zig:sum"
+```
+
+This function accepts Python [arrays](https://docs.python.org/3/library/array.html#module-array), Numpy arrays, or any other buffer protocol implementation.
+
+```python
+import numpy as np
```

Comment:
Would be good to import this from test_buffers.py

Reply:
Done